package-lock.json

```diff
@@ -5944,8 +5958,17 @@
       "requires": {
         "nan": "^2.2.1",
         "postis": "^2.2.0",
-        "prebuild-install": "^2.4.1",
-        "robotjs": "github:jitsi/robotjs#5cc469f655669e728b61271c2c57ce97d62976fd"
+        "prebuild-install": "^2.4.1"
+      },
+      "dependencies": {
+        "robotjs": {
```

Comment:
I think the Travis build might be breaking because if this `package-lock.json` change. Since `package.json` does not have any change the update in lock file might be due to version mismatch of `npm` or `node`.

Travis uses node `v8.16.0` and `npm 6.4.1`.

Can you tell the version of node and npm you are using?

Reply:
Hi, i'm using 
Node: 10.15.3 
npm:   6.4.1 (same)